app/connectors/parsers/DeleteInsurancePoliciesParser.scala

```diff
@@ -43,6 +43,9 @@ object DeleteInsurancePoliciesParser extends APIParser with Logging {
       case BAD_REQUEST | NOT_FOUND =>
         pagerDutyLog(FOURXX_RESPONSE_FROM_API, logMessage(response))
         handleAPIError(response)
+      case UNPROCESSABLE_ENTITY =>
```

Comment:
Can we please cross check if get `UNPROCESSABLE_ENTITY` for Tax year not supported?

Reply:
corrected the errors to include override if MTD error codes don't match downstream error codes (table on confluence page for delete API)